Automattic/simplenote-ios
# Add dismiss handle to Information Sheet

### Fix
This PR continues with the design changes in issue #1061. Specifically this PR adds in a visual element to alert the user that they can dismiss the information sheet by dragging down on the top of the information sheet element.

The drag bar element will not appear on iPad and should look a bit like this:

<img src="https://cdn-std.droplr.net/files/acc_593859/WpyBbZ" />

### Test

1. Open up Simplenote on an iPhone and go to a Note > Information sheet, confirm that the drag bar appears
2. Open up Simplenote again on an iPad and go to a note > Information sheet, confirm there is no drag bar

### Review
One developer and one designer are required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
